spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/ConfigServiceOperations.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.config;
+
+import com.microsoft.azure.spring.cloud.config.domain.KeyValueItem;
+import org.springframework.lang.Nullable;
+
+import javax.validation.constraints.NotEmpty;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Operations to access certain Azure Config Service configuration store.
+ */
+public interface ConfigServiceOperations {
+    /**
+     * Find a config key-value item with given {@code keyName} and {@link Nullable} {@code label}
+     *
+     * @param keyName name of the key
+     * @param label   label of the key, can be null
+     * @return {@link Optional<KeyValueItem>}, if key does not exist, return {@link Optional#empty()}.
+     */
+    Optional<KeyValueItem> getKey(String keyName, @Nullable String label);
```

Comment:
Do we really need it?

Reply:
Probably no requirement currently, will remove.